public/e2e-test/scenarios/smoke.test.ts

```diff
@@ -0,0 +1,85 @@
+import { Browser, Page, Target } from 'puppeteer-core';
+
+import { e2eScenario } from 'e2e-test/core/scenario';
+import { addDataSourcePage } from 'e2e-test/pages/datasources/addDataSourcePage';
+import { editDataSourcePage } from 'e2e-test/pages/datasources/editDataSourcePage';
+import { dataSourcesPage } from 'e2e-test/pages/datasources/dataSources';
+import { createDashboardPage } from 'e2e-test/pages/dashboards/createDashboardPage';
+import { saveDashboardModal } from 'e2e-test/pages/dashboards/saveDashboardModal';
+import { dashboardsPageFactory } from 'e2e-test/pages/dashboards/dashboardsPage';
+import { panel } from 'e2e-test/pages/panels/panel';
+import { editPanelPage } from 'e2e-test/pages/panels/editPanel';
+import { constants } from 'e2e-test/core/constants';
+import { sharePanelModal } from 'e2e-test/pages/panels/sharePanelModal';
+import { takeScreenShot, compareScreenShots } from 'e2e-test/core/images';
+
+e2eScenario(
+  'Login scenario, create test data source, dashboard, panel, and export scenario',
+  'should pass',
+  async (browser: Browser, page: Page) => {
+    // Add TestData DB
+    await addDataSourcePage.init(page);
```

Comment:
It's a bit bothersome that we introduce additional test abstractions - there is a danger of accidentally writing our own test framework haha.

But in such environment I suppose something of the sort is inevitable. But just in case (we probably talked about it but a lot time has passed), we did consider visual test frameworks right?

Reply:
Good point! I think this will have to do for now, I think it's still light weight WDYT @markelog?

Reply:
Make sense to me! :)